src/store/modules/getters/staticData.js

```diff
@@ -40,3 +40,5 @@ export const getNFTClassIdSorter = (_, getters) => classIds => {
   });
   return sorted;
 };
+
+export const LIKEPriceInUSD = state => state.LIKEPriceInUSD;
```

Comment:
```suggestion
export const LIKEPriceInUSD = state => state.LIKEPriceInUSD || ROUGH_LIKE_TO_USD_PRICE;
```

https://github.com/likecoin/liker-land/blob/af8403b5fddbe408b59d9994ddad3833d99aa687/src/constant/index.js#L106